### ISSUE TITLE
No stepsize here.

### ISSUE THREAD

**Issue by foTok** [hazrmard/QLearning#1]
https://github.com/hazrmard/QLearning/blob/f9409733351fc01a55e9986a4b1df19851c03d10/models/fuel_tanks.py#L81

Hi, if you just set the threshold as 10, I think it means that we just simulate 1 time step. However, in the following code, you added step size to simulate how fuel flows between tanks. By default, "stepsize=1", I think the program works well. But I am afraid if you set stepsize as some values else, there would be problem.

**Comment by hazrmard** [comment]
Good catch. I patched that issue (by scaling standard demand of `10` by the `stepsize`) and added some comments explaining the model in commit b46ee64974c06481b42f1dd76e3b8700c0e15321.

You can pull the repo to apply those changes. Or simply copy-paste the `models/fuel_tanks.py` file.
